packages/flutter_markdown/README.md

```diff
@@ -17,10 +17,10 @@ containing a rich text representation.
 the Markdown into an abstract syntax tree (AST). The nodes of the AST are an
 HTML representation of the Markdown data.
 
-## Flutter Isn't a HTML Renderer
+## Flutter Isn't an HTML Renderer
```

Comment:
I'd argue the H in HTML is sounded, and thus a is correct in this position per https://owl.purdue.edu/owl/general_writing/grammar/articles_a_versus_an.html

Hey @sfshaza2 or @atsansone wanna tie break?

Reply:
In the link, there is no instance of "a h", but twice "an h"

> Exceptions
Use "an" before a silent or unsounded "h." Because the "h" does not have any phonetic representation or audible sound, the sound that follows the article is a vowel; consequently, "an" is used.
an honorable peace
an honest error

Reply:
> I'd argue the H in HTML is sounded

For an initialism it doesn't really make sense to say a letter is sounded or unsounded; the sound of the letters isn't used at all.

The deciding factor is the pronunciation of the letter H as a word. Since Flutter style uses US English spelling, we should spell this based on prevailing US pronunciation (which starts with an "ay"/"ai" sound), making "an" correct.